Platform/MeteorlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -425,7 +425,7 @@ UpdateFspConfig (
   CopyMem(Fspmcfg->DqMapCpu2DramMc1Ch2, MemCfgData->DqMapCpu2DramMc1Ch2, sizeof(MemCfgData->DqMapCpu2DramMc1Ch2));
   CopyMem(Fspmcfg->DqMapCpu2DramMc1Ch3, MemCfgData->DqMapCpu2DramMc1Ch3, sizeof(MemCfgData->DqMapCpu2DramMc1Ch3));
 
-  Fspmcfg->ActiveSocNorthAtomCoreCount                          = 0xFF;
+  Fspmcfg->ActiveSocNorthAtomCoreCount                          = 0x2;
```

Comment:
Please double check this. I believe MTL silicon doesn't always has 2 small core.  If you gave a specific value the system will reboot if the core count doesn't match this value.  Often value 0xFF (active all small core) is a better value.

Reply:
Alright noted, have changed the value to 0xFF